lcdb_test_data/Snakefile

```diff
@@ -1,19 +1,31 @@
 import os
 from textwrap import dedent
 
+# required to avoid near-simultaneous timestamps that confuse snakemake
 shell.prefix('sleep 2;')
 
+
 rule all:
-    input: expand('samples/sample{n}/sample{n}.bam', n=range(1, 5))
+    input:
+        expand(
+            'samples/sample{n}/sample{n}.{paired}.bam',
+            n=range(1, 5), paired=['paired', 'single']
+        )
+
 
+# ----------------------------------------------------------------------------
+# Create a BED file that will be used to subset GTF and FASTA files
 rule limits:
     output: temp('LIMIT.bed')
     shell:
         'echo "2L	0	80000	2L" > {output}'
 
+
+# ----------------------------------------------------------------------------
+# Download FlyBase GTF, so
 rule prep_gtf:
     input: rules.limits.output
```

Comment:
Have not seen this before. I am guessing it is taking the output from the rule limits. Is this correct?